packages/docker/src/executors/release-publish/release-publish.impl.ts

```diff
@@ -0,0 +1,93 @@
+import { ExecutorContext, ProjectGraphProjectNode, logger } from '@nx/devkit';
+import { execSync } from 'child_process';
+import {
+  DockerReleasePublishSchema,
+  NormalizedDockerReleasePublishSchema,
+} from './schema';
+import { join } from 'path';
+import { existsSync, readFileSync } from 'fs';
+
+export default async function dockerReleasePublish(
+  schema: DockerReleasePublishSchema,
+  context: ExecutorContext
+) {
+  logger.warn(
+    `Docker support is experimental. Breaking changes may occur and not adhere to semver versioning.`
```

Comment:
Let's only warn once maybe during nx-release rather than once for every single project.